Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     killbill-cybersource (4.0.11)
+      accent_to_ascii (~> 0.1.1)
```

Comment:
It looks like it's just a mapping table? Could we hardcode it in the plugin instead of pulling another gem?